frontend/src/stores/ui.ts

```diff
@@ -133,16 +133,30 @@ export const useUiStore = defineStore("ui", {
      * Given a list of key value pairs of UI state parameters and their states,
      * populate the store state to match the cookie.
      *
+     * Since the cookies are passed through `JSON.parse()`, they can have the
+     * wrong types. This function resets cookies to the default value if their
+     * type is incorrect.
+     *
      * @param cookies - mapping of UI state parameters and their states.
      */
     initFromCookies(cookies: OpenverseCookieState) {
-      this.updateBreakpoint(cookies.uiBreakpoint ?? this.breakpoint)
-      this.isFilterDismissed = cookies.uiIsFilterDismissed ?? false
-      this.isMobileUa = cookies.uiIsMobileUa ?? false
+      let breakpoint = this.breakpoint
+      if (
+        cookies.uiBreakpoint &&
+        Object.keys(ALL_SCREEN_SIZES).includes(cookies.uiBreakpoint)
+      )
+        breakpoint = cookies.uiBreakpoint
+      this.updateBreakpoint(breakpoint)
```

Comment:
```suggestion
        breakpoint = cookies.uiBreakpoint
      this.updateBreakpoint(breakpoint)
```

Could you add curly braces to the `if` statements here, and perhaps even some line breaks between blocks? these lines in particular seem particularly difficult for folks to parse.

Reply:
Done in 1d3326a.